otelcollector/shared/configmap/mp/configmapparser.go

```diff
@@ -239,18 +238,17 @@ func Configmapparser() {
 		}
 
 		// Source prom_config_validator_env_var
-		cmd := exec.Command("bash", "-c", "source /opt/microsoft/prom_config_validator_env_var && env")
-		if err := cmd.Run(); err != nil {
-			shared.EchoError("Error sourcing env file:" + err.Error())
-			return
-		}
-
-		// Source envvars.env
-		cmd = exec.Command("bash", "-c", "source /opt/envvars.env && env")
-		if err := cmd.Run(); err != nil {
-			shared.EchoError("Error sourcing envvars.env:" + err.Error())
-			return
-		}
+		filename := "/opt/microsoft/prom_config_validator_env_var"
```

Comment:
Kaveesh - shouldn't we check if os is linux or windows and set env for both ?

Reply:
The function `shared.SetEnvVarsFromFile(filename)` works for both linux and windows, hence we don't need to check the os before calling it.

I validated this by going in both the linux and windows pods and the environment variables were set correctly from the file.